File: Easy/771 - Jewels and Stones/771Soln.rb
# frozen_string_literal: true

# @param {String} j
# @param {String} s
# @return {Integer}
def num_jewels_in_stones(j, s)
  count = 0
  j.split('').each do |jewel|
    s.split('').each do |stone|
      jewel == stone ? count += 1 : count
    end
  end

  count
end

File: Easy/771 - Jewels and Stones/771Soln.js
'use strict'

/**
 * @param {string} J
 * @param {string} S
 * @return {number}
 */
const numJewelsInStones = function(J, S) {
    let count = 0
    J.split('').forEach(jewel => {
        S.split('').forEach(stone => {
            if (stone === jewel) { count++ }
        })
    })

    return count
}
